src/v3/src/util/designTokens.ts

```diff
@@ -0,0 +1,237 @@
+
+  // Spacing
+  Spacing0: "0",
+  Spacing1: "0.28571429rem",
+  Spacing2: "0.57142857rem",
+  Spacing3: "0.85714286rem",
+  Spacing4: "1.14285714rem",
+  Spacing5: "1.71428571rem",
+  Spacing6: "2.28571429rem",
+  Spacing7: "2.85714286rem",
+  Spacing8: "3.42857143rem",
+  Spacing9: "4rem",
+
+  // Transition
+  TransitionDurationMain: "100ms",
+  TransitionTimingMain: "linear",
+
+  // TypographyColor
+  TypographyColorBody: "#272727", // theme.palette.text.primary
+  TypographyColorHeading: "#272727", // theme.typography.h1.color
+  TypographyColorInverse: "#ffffff",
+  TypographyColorSupport: "#4b4b4b",
+  TypographyColorSubordinate: "#6e6e6e", // theme.typography.caption.color
+  TypographyColorDisabled: "#aeaeae", // theme.palette.text.disabled
+  TypographyColorAction: "#4c64e1",
+  TypographyColorDanger: "#d92300",
+  TypographyColorSuccess: "#197f48",
+  TypographyColorWarning: "#966603",
+
+  // TypographyFamily
+  TypographyFamilyBody: "'Inter', '-apple-system', 'BlinkMacSystemFont', 'Segoe UI', 'Roboto', 'Oxygen-Sans', 'Ubuntu', 'Cantarell', 'Helvetica Neue', sans-serif", // theme.typography.fontFamily
+  TypographyFamilyHeading: "'Aeonik', 'Inter', '-apple-system', 'BlinkMacSystemFont', 'Segoe UI', 'Roboto', 'Oxygen-Sans', 'Ubuntu', 'Cantarell', 'Helvetica Neue', sans-serif", // theme.typography.h1.fontFamily
+  TypographyFamilyButton: "'Aeonik', 'Inter', '-apple-system', 'BlinkMacSystemFont', 'Segoe UI', 'Roboto', 'Oxygen-Sans', 'Ubuntu', 'Cantarell', 'Helvetica Neue', sans-serif", // theme.typography.button.fontFamily
+  TypographyFamilyMono: "'Roboto Mono', 'SFMono-Regular', Consolas, 'Liberation Mono', Menlo, Courier, monospace", // theme.typography.kbd.fontFamily
+
+  // TypographyScale
+  TypographyScale0: "0.857rem",
+  TypographyScale1: "1rem",
+  TypographyScale2: "1.143rem",
+  TypographyScale3: "1.286rem",
+  TypographyScale4: "1.429rem",
+  TypographyScale5: "1.571rem",
+  TypographyScale6: "1.786rem",
+  TypographyScale7: "2rem",
+  TypographyScale8: "2.286rem",
+  TypographyScale9: "2.571rem",
+  TypographyScale10: "2.857rem",
+  TypographyScale11: "3.214rem",
+  TypographyScale12: "3.643rem",
+
+  // TypographySize
+  TypographySizeBase: "87.5%", // theme.typography.body1.fontSize
+  TypographySizeSubordinate: "0.857rem", // theme.typography.caption.fontSize
+  TypographySizeBody: "1rem", // theme.typography.body1.fontSize
+  TypographySizeHeading6: "1.143rem", // theme.typography.h6.fontSize
+  TypographySizeHeading5: "1.286rem", // theme.typography.h5.fontSize
+  TypographySizeHeading4: "1.571rem", // theme.typography.h4.fontSize
+  TypographySizeHeading3: "2.000rem", // theme.typography.h3.fontSize
+  TypographySizeHeading2: "2.286rem", // theme.typography.h2.fontSize
+  TypographySizeHeading1: "2.571rem", // theme.typography.h1.fontSize
+
+  // TypographyStyle
+  TypographyStyleNormal: "normal", // theme.typography.body1.fontStyle
```

Comment:
Is this token supposed to be set in `theme.ts` because it is not currently being set. Also, if the purpose of this token is to allow users to customize the `font-style` property of body text, would a name like `TypographyStyleBody` make more sense and then users could set it to normal, italic, etc.?